### DISCUSSION TITLE
Z-stage part of the xyz-stage

### DISCUSSION THREAD

**Discussion by flyingpinguu** [bionanoimaging/UC2-GIT#99] [General]
Hello there,

I am currently building the xyz-stage for a flourescence microscope build. I have the parts printed and the electronics pretty much done too, but I have some trouble understanding how the objective will be placed and how the flexure piece can move it in the z direction. 
https://github.com/openUC2/UC2-GIT/blob/master/CAD/ASSEMBLY_XYZ-STAGE_v0/DOCUMENTS/TUT_Assembly_XYZ_Stage.pdf
The instructions are helpful, but there are no pictures available where I can see how this is realized. Maybe there is a better picture of how the z-part works of the xyz-stage? This is where I'm at:
Thanks a lot!
![WhatsApp Image 2022-09-30 at 13 26 43](https://user-images.githubusercontent.com/106193486/193260140-0f09ec71-8b1c-4508-8cb0-083c09b54267.jpeg)

**Comment by beniroquai** [comment]
Wow! I'm really impressed that you managed to recreate the XYZ-stage! Well done! :-) 
Would you mind posting the holder for the objective lens you printed? There must be a whole for a M3 screw that slides into the "rail" of the z-translator :)

**Reply by flyingpinguu** [reply]
![WhatsApp Image 2022-10-04 at 13 18 51](https://user-images.githubusercontent.com/106193486/193806184-4b4c0d1a-4868-4cac-a568-e125f0f07d31.jpeg)
I suppose you mean this piece? It fits on the rail, I'm just not sure how to fixate it and make the stepper motor move it. :)

**Comment by beniroquai** [comment]
This little kink is supposed to fit an m3 screw. So in general you have to
slide this mechanism in than fixated using the screw by screen it's such as
that it's getting pressed against the rail. Does  this makes sense to you?

However I must admit that this is a rather all design and I'm not fully
able to support this anymore. We do have newer versions of the stages and
also XY stages. If you search through the openuc2 to organisational you
will find it. Just reach out if you have any additional questions

**Reply by flyingpinguu** [reply]
Alright, I understand now how it can be fixated with a screw, but I'm still not sure how it's going to move up and down. If I move the flexing part on the other side of the motor (the part that is conncted to the gears), the rail moves by maybe 0.5-1mm. Is that enough?
Also, which stage would you recommend switching to if I decide that, the Delta-Stage or something else?
Thanks again :)

**Reply by beniroquai** [reply]
Yes! That's the main idea => Reduction :) 
We wanted to have a precise motion along Z. But again, I was never really happy with the 3D printed stages (e.g. the openflexure). 

Are you talking about a z-stage you want to switch to or also xy? 

You can find more here:
https://github.com/openUC2/UC2-Zstage
and here:
https://github.com/openUC2/UC2-Motorized-XY-Table

Ping me before you want to build it ;-)

Was the problem with Imswitch resolved?
